osf_pigeon/pigeon.py

```diff
@@ -395,7 +406,7 @@ async def archive(guid):
         # await first to check if withdrawn
         metadata = await get_registration_metadata(guid, os.path.join(temp_dir, "bag"), "registration.json")
         tasks = [
-            write_datacite_metadata(guid, temp_dir, metadata),
+            #write_datacite_metadata(guid, temp_dir, metadata),
```

Comment:
Should this be commented out?

Reply:
No ug, I just did that for testing locally, sorry.